docs/reference/installation.rst

```diff
@@ -23,11 +23,7 @@ Next, add the dependent bundles:
     composer update
 
 
-Now, add the new `SonataNotificationBundle` Bundle to ``bundles.php`` file:
-
-.. code-block:: php
-
-    <?php
+Now, add the new `SonataNotificationBundle` Bundle to ``bundles.php`` file::
```

Comment:
should the bundle name be in double ticks?

Reply:
done